packages/cashier/src/stores/__tests__/withdraw-store.spec.ts

```diff
@@ -1,7 +1,8 @@
 import { isMobile, validNumber } from '@deriv/shared';
 import WithdrawStore from '../withdraw-store';
 import { configure } from 'mobx';
-import { TWebSocket, TRootStore } from 'Types';
+import { TWebSocket } from '../../types';
+import { mockStore, TStores } from '@deriv/stores';
```

Comment:
```suggestion
import { mockStore } from '@deriv/stores';
```
@rostislav-deriv Use `TRootStore` instead 🙏🏻

Reply:
Aren't we supposed to use `TStores` in tests? 🤔 After all, `mockStore` returns exactly that